website/content/en/_index.html

```diff
@@ -0,0 +1,81 @@
++++
+title = "Cloud Operations Sandbox"
+linkTitle = "Cloud Operations Sandbox"
+
++++
+
+{{< blocks/cover title="Welcome to Goldydocs: A Docsy Example Project!" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{ relref . "/docs" }}">
+		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/google/docsy-example">
+		Download <i class="fab fa-github ml-2 "></i>
+	</a>
+	<p class="lead mt-5">Porridge temperature assessment - in the cloud!</p>
```

Comment:
What is this text about?

Reply:
Actually this is an unused file, I'll delete it!